opensearchpy/_async/client/nodes.pyi

```diff
@@ -79,7 +88,6 @@ class NodesClient(NamespacedClient):
         fields: Optional[Any] = ...,
         groups: Optional[Any] = ...,
         include_segment_file_sizes: Optional[Any] = ...,
-        include_unloaded_segments: Optional[Any] = ...,
```

Comment:
Are we breaking users by removing this?

Reply:
https://github.com/opensearch-project/opensearch-py/pull/514#issue-1918245776

Reply:
We are not breaking users because the parameter is throwing error
```

{
"type": "illegal_argument_exception",
"reason": "request [/_nodes/stats] contains unrecognized parameter: [include_unloaded_segments]"
}
```

Reply:
Thats weird, the parameter is present in the server https://github.com/opensearch-project/OpenSearch/blob/main/server/src/main/java/org/opensearch/rest/action/cat/RestIndicesAction.java#L136

Reply:
I'm not certain, Vacha. I'm not very familiar with the OpenSearch server code. Do we need to examine this file or action?

https://github.com/opensearch-project/OpenSearch/blob/main/server/src/main/java/org/opensearch/rest/action/admin/cluster/RestNodesStatsAction.java#L63

Reply:
Ah you are absolutely right! For nodes stats it does not exist.